core-processor/src/precharge.rs

```diff
@@ -145,24 +186,6 @@ impl<'a> GasPrecharger<'a> {
                 .cost_for_with_bytes(self.costs.instrumentation_per_byte, original_code_len_bytes),
         )
     }
-
-    /// Charge gas for pages and checks that there is enough gas for that.
-    /// Returns size of wasm memory buffer which must be created in execution environment.
-    pub fn charge_gas_for_pages(
```

Comment:
are we on purpose no longer charge for static pages allocation?

Reply:
Yes, I think,  we are 
cc @grishasobol

Reply:
Yes